js/src/evaluation/_runner.ts

```diff
@@ -133,13 +145,17 @@ interface _ExperimentManagerArgs {
   runs?: AsyncGenerator<Run>;
   evaluationResults?: AsyncGenerator<EvaluationResults>;
   summaryResults?: AsyncGenerator<
-    (runsArray: Run[]) => AsyncGenerator<EvaluationResults, any, unknown>,
+    (
```

Comment:
This is a weird typing but I guess fine

Reply:
Ah yeah - can I just make evaluationResults an optional param? Or is there a better solution to make this non breaking?